models/user.js

```diff
@@ -42,7 +42,27 @@ const UserSchema = mongoose.Schema({
   users_following: {
     type: [String],
     default: []
-  }
+  },
+  notifications: {
+    type: [{
+      user_from: String,
```

Comment:
I might rename to `from_user`, as this seems to be used below as:
`notification.from_user` is the user which sent the notification, which reads better than `notification.user_from`
Similarly, `from_username` (or with underscore if that is consistent elsewhere)

Reply:
I might even use a more generic term like "source" and "source_description" in case of cases where the system generates notifications, etc.